### ISSUE TITLE
buildProtectedAttributeName in SplittingModule class

### ISSUE THREAD

**Issue by tchevallier** [clarus-proxy/splitting-module#3]
Hi Alberto, I noticed that there has been a change in the buildProtectedAttributeName method in the SplittingModule class. Unfortunately the new value returned prevents our use case from properly functioning. I created a dedicated branch to fix this ( feature/pgsql-integration ), but I did not commit to the develop/master branch as there might be some reason on your side for changing it. Could you please have a look ?

**Comment by ablancoj** [comment]
Yes, these were changes needed to execute the demo on the local side and
thus might cause problems in the integrated module.

**Comment by tchevallier** [comment]
OK thanks, then it would be preferrable to have it the other way round (i.e. the code for integrated functioning in the master branch , and the patch for standalone demo in a dedicated branch)
